dockerfile-image-update/src/main/java/com/salesforce/dockerfileimageupdate/utils/PullRequests.java

```diff
@@ -41,5 +49,32 @@ public void prepareToCreate(final Namespace ns,
         }
         ResultsProcessor.processResults(skippedRepos, exceptions, log);
     }
+
+    /**
+     * Check if the repository is onboarded to Renovate. The signal we are looking for are
+     * (1) The presence of a file where renovate configurations are stored in the repository
+     * (2) Ensuring that the file does not have the key "enabled" set to "false"
+     * @param filePaths the list that contains all the names of the files that we are searching for in the repo
+     * @param fork A GitHubContentToProcess object that contains the fork repository that is under process
+     * @return true if the file is found in the path specified and is not disabled, false otherwise
+     */
+    private boolean isRenovateEnabled(List<String> filePaths, GitHubContentToProcess fork) throws IOException {
+        for (String filePath : filePaths) {
+            try {
+                GHContent fileContent = fork.getParent().getFileContent(filePath);
+                InputStream is = fileContent.read();
+                BufferedReader bufferedReader = new BufferedReader(new InputStreamReader(is));
+                JSONTokener tokener = new JSONTokener(bufferedReader);
+                JSONObject json = new JSONObject(tokener);
+                is.close();
+                bufferedReader.close();
```

Comment:
You might want to do this in a try-with-resources statement, see https://stackoverflow.com/a/56151320 .  If `is.close` throws an exception then the close on buffered reader won't happen.